control/veda_deliver.py

```diff
@@ -526,20 +530,30 @@ def _BOTO_MULTIPART(self):
         return True
 
     def _CIELO24_UPLOAD(self):
-        if self.video_query.inst_class.c24_proc is False:
+        """
+        Cielo24 transcription flow.
+        """
+        # TODO! ask about mobile_override
+        if self.encode_profile != 'desktop_mp4':
             return None
 
-        if self.video_query.inst_class.mobile_override is False:
```

Comment:
@muhammad-ammar I think that's good. It was a legacy safety check from the days before HLS. 👍